### PR TITLE
Remove nightlies

### PR DESCRIPTION
**Pull request by zjhuntin**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [x] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by zjhuntin** [comment]
I don't think we should remove those from the releasers conf yet, though I do not know when that should happen.

**Comment by ehelms** [comment]
@zjhuntin The releasers should get updated with this PR or in a follow up. No reason to keep them around in that form, and potential for errors in the future.

**Comment by akofink** [comment]
LGTM

File: rel-eng/releasers.conf [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
 
 [koji-katello-el6]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel6
+autobuild_tags = katello-3.6-rhel6
 
 [koji-katello-jenkins-el6]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel6 katello-thirdparty-rhel6
+autobuild_tags = katello-3.6-rhel6 katello-thirdparty-rhel6
```
Comment (ehelms) [inline rel-eng/releasers.conf]: This needs an update to `katello-3.6-thirdparty-rhel6`

File: rel-eng/releasers.conf [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
 
 [koji-katello-el7]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel7 katello-thirdparty-rhel7
+autobuild_tags = katello-3.6-rhel7 katello-thirdparty-rhel7
 
 [koji-katello-jenkins-el7]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel7 katello-thirdparty-rhel7
+autobuild_tags = katello-3.6-rhel7 katello-thirdparty-rhel7
```
Comment (ehelms) [inline rel-eng/releasers.conf]: This needs an update to `katello-3.6-thirdparty-rhel7`

File: rel-eng/releasers.conf [hunk 3]
```diff
@@ -1,61 +1,61 @@
...
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora25 katello-nightly-fedora26 katello-thirdparty-rhel7
+autobuild_tags = katello-3.6-rhel5 katello-3.6-rhel6 katello-3.6-rhel7 katello-3.6-fedora25 katello-3.6-fedora26 katello-thirdparty-rhel7
 builder = tito.builder.FetchBuilder
 builder.jenkins_url = http://ci.theforeman.org
 
 [koji-katello-el7]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel7 katello-thirdparty-rhel7
+autobuild_tags = katello-3.6-rhel7 katello-thirdparty-rhel7
```
Comment (ehelms) [inline rel-eng/releasers.conf]: This needs an update to `katello-3.6-thirdparty-rhel6`

File: rel-eng/releasers.conf [hunk 4]
```diff
@@ -1,61 +1,61 @@
...
 
 [koji-katello-client]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora25 katello-nightly-fedora26 katello-thirdparty-rhel7
+autobuild_tags = katello-3.6-rhel5 katello-3.6-rhel6 katello-3.6-rhel7 katello-3.6-fedora25 katello-3.6-fedora26 katello-thirdparty-rhel7
 
 [koji-katello-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel5 katello-nightly-rhel6 katello-nightly-rhel7 katello-nightly-fedora25 katello-nightly-fedora26 katello-thirdparty-rhel7
+autobuild_tags = katello-3.6-rhel5 katello-3.6-rhel6 katello-3.6-rhel7 katello-3.6-fedora25 katello-3.6-fedora26 katello-thirdparty-rhel7
```
Comment (ehelms) [inline rel-eng/releasers.conf]: This needs an update to `katello-3.6-thirdparty-rhel6`

**Pull request merged**